workers/unity/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveBlittableSingularGameObjectComponentDispatcher.cs

```diff
@@ -0,0 +1,162 @@
+// ===========
+// DO NOT EDIT - this file is automatically regenerated.
+// ===========
+
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Entities;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+
+namespace Generated.Improbable.Gdk.Tests
+{
+    public partial class ExhaustiveBlittableSingular
+    {
+        internal class GameObjectComponentDispatcher : GameObjectComponentDispatcherBase
+        {
+            public override ComponentType[] ComponentAddedComponentTypes => new ComponentType[]
+            {
+                ComponentType.ReadOnly<ComponentAdded<SpatialOSExhaustiveBlittableSingular>>(), ComponentType.ReadOnly<GameObjectReference>()
+            };
+
+            public override ComponentType[] ComponentRemovedComponentTypes => new ComponentType[]
+            {
+                ComponentType.ReadOnly<ComponentRemoved<SpatialOSExhaustiveBlittableSingular>>(), ComponentType.ReadOnly<GameObjectReference>()
+            };
+
+            public override ComponentType[] AuthoritiesChangedComponentTypes => new ComponentType[]
+            {
+                ComponentType.ReadOnly<AuthorityChanges<SpatialOSExhaustiveBlittableSingular>>(), ComponentType.ReadOnly<GameObjectReference>()
+            };
+
+            public override ComponentType[] ComponentsUpdatedComponentTypes => new ComponentType[]
+            {
+                ComponentType.ReadOnly<SpatialOSExhaustiveBlittableSingular.ReceivedUpdates>(), ComponentType.ReadOnly<GameObjectReference>()
+            };
+
+            public override ComponentType[][] EventsReceivedComponentTypeArrays => new ComponentType[][]
```

Comment:
No but it all contributes to the gc limit before you do a big clean. Not a big deal but eventually it would be nice to not allocate anything we don't need.

Reply:
Created https://improbableio.atlassian.net/browse/UTY-1068 to track this.